riocli/device/execute.py

```diff
@@ -28,15 +28,19 @@
     help_options_color=Colors.GREEN,
 )
 @click.option('--user', default='root')
+@click.option('--timeout', default=300)
 @click.option('--shell', default='/bin/bash')
+@click.option('--run-async', is_flag=True, default=True, help="Run the command in the background.")
 @click.argument('device-name', type=str)
 @click.argument('command', nargs=-1)
 @name_to_guid
 def execute_command(
         device_name: str,
         device_guid: str,
         user: str,
+        timeout: int,
         shell: str,
+        run_async: bool,
```

Comment:
todo: Please pull from devel and update the docstring to explain the new option.

Reply:
Done